best_fit_fluxes2escher_fluxes_csv.sh

```diff
@@ -3,7 +3,7 @@
 echo "ID,Avg" > $2
 FOURTHCOL=`head -n 1 $1 | gawk -vFPAT='[^,]*|"[^"]*"' '{ print $4 }'`
 if [ $FOURTHCOL == "Value" ]; then
-   # we are looking at an iso2flux 0.6.1 file
+   # we are looking at an iso2flux 0.6.1 or newer file
```

Comment:
I would suggest then being explicit and say something like 0.6.1  to 0.7... who knows what will come first!